modules/global-search/src/main/java/com/ibm/cloud/platform_services/global_search/v2/model/ScanResult.java

```diff
@@ -24,9 +24,11 @@ public class ScanResult extends GenericModel {
 
   @SerializedName("search_cursor")
   protected String searchCursor;
-  protected Long limit;
+  protected Double limit;
```

Comment:
This is the result in Java of the ScanResult.limit property being defined as "number" instead of "integer".

Reply:
At this question can answer Michele Crudele.